src/main/java/org/openrewrite/java/testing/cleanup/AssertLiteralBooleanToFailRecipe.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Moderne Source Available License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://docs.moderne.io/licensing/moderne-source-available-license
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.cleanup;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.JavaVisitor;
+import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaType;
+
+
+public class AssertLiteralBooleanToFailRecipe extends Recipe {
```

Comment:
What made it that this could no longer be a Refaster recipe? As that's not clear immediately, and Refaster is preferred where possible.

Reply:
I'd view it as merely adding a new Refaster recipe alongside the original one we already had, and then including both in a declarative recipe.

Reply:
Having now seen that converting the other recipe to refaster is not possible.
And knowing that maintaining the recipe ID is not important.
I'd suggest merging these two recipes into a single recipe which handles all assertTrue/False(<literal>) cases.

Any thoughts @timtebeek?